src/Nav/Nav.tsx

```diff
@@ -130,6 +130,7 @@ const Nav: FC = () => {
 					className={classes.title}
 					role="heading"
 					variant="h4"
+					component="h1"
```

Comment:
We do not need to specify that an `<h1>` tag is used here

```suggestion
```

Reply:
I re-added this cause I didn't catch it was removed from an earlier PR I had. It's an accessibility rule that is easy to cover. https://dequeuniversity.com/rules/axe/4.2/page-has-heading-one

Reply:
Also you can see I was originally testing for it but cypress was not ran when the line of code was removed, see the deleted page simplePage.spec.js

Reply:
Hmmm - requiring a page to have a level one heading seem arbitrary, but I suppose if it's for accessibility, we can just style the h1 differently. Thanks for the explanation